CHANGELOG.md

```diff
@@ -13,6 +13,10 @@
 - Started a changelog.
 - Updated pull request template
 
+### ♻️ Refactoring
```

Comment:
We aren't including a separate "Refactoring" section; instead, please include this entry under "Internal" above

Reply:
You should then delete the header (and blank line above it)